pkg/kubectl/cmd/util/openapi/validation/validation_test.go

```diff
@@ -335,4 +335,63 @@ items:
 
 		Expect(err).To(BeNil())
 	})
+
+	It("fails because apiVersion is not provided", func() {
+		err := validator.ValidateBytes([]byte(`
+kind: Pod
+metadata:
+  name: name
+spec:
+  containers:
+  - name: name
+    image: image
+`))
+		Expect(err.Error()).To(Equal("apiVersion not set"))
```

Comment:
I'm not super fan of testing errors by string, can you look at how it's done for other errors?

Reply:
@apelisse hi, thanks much for the quick review and suggestions :-). Here, I actually used one of the current tests as reference - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/openapi/validation/validation_test.go#L214  but let me see if I can use other tests. Thanks!